menus/menus.py

```diff
@@ -13,14 +11,14 @@
 
 class OptionsMenu(Screen):
     @classmethod
-    def start(cls, screen: pygame.Surface, close: Callable[[], None]) -> None:
-
-        menu_surface = pygame.Surface(
-            size=config.RESOLUTION, flags=pygame.SRCALPHA, depth=32
-        )
-        menu_surface.convert_alpha()
-        menu_surface.fill(pygame.Color(0, 0, 0, 200))
-        screen.blit(source=menu_surface, dest=(0, 0))
+    def start(cls, screen: pygame.Surface, close: Callable[[], None], on_top = True) -> None:
+        if on_top:
+            menu_surface = pygame.Surface(
+                size=config.RESOLUTION, flags=pygame.SRCALPHA, depth=32
+            )
+            menu_surface.convert_alpha()
+            menu_surface.fill(pygame.Color(0, 0, 0, 200))
+            screen.blit(source=menu_surface, dest=(0, 0))
 
         def visuals_button_click():
             pass
```

Comment:
Did you just forget to write this function or is this an abstract class and this should be implemented somewhere else?
If this is an abstract class:

```suggestion
            raise NotImplementedError()
```